content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -368,15 +367,12 @@ to `Retain`, including cases where you are reusing an existing PV.
 Support for expanding PersistentVolumeClaims (PVCs) is enabled by default. You can expand
 the following types of volumes:
```

Comment:
I think that we should just remove all these examples "You can expand
the following types of volumes:" as all these are either already removed or deprecated.

Reply:
We usually provide documentation for deprecated features, right up to the point that they are actually removed.